js/src/sheet.ts

```diff
@@ -68,7 +68,7 @@ let SheetModel = widgets.DOMWidgetModel.extend({
     initialize : function () {
         SheetModel.__super__.initialize.apply(this, arguments);
         this.data = [[]];
-        this.update_data_grid();
+        this.update_data_grid(false);
```

Comment:
This should improve performances a bit ;) Not triggering "data_change" event twice at initialization